src/workloads/docs/MonotonicSingleLoader.yml

```diff
@@ -1,7 +1,7 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/sharding"
 Description: |
-  TODO: TIG-3320
+  Loads a large set of documents with a random value assinged to `field`.
 
```

Comment:
There's a keywords field we could also add to these workloads that we are starting to parse in our workload documentation. See for example: https://github.com/mongodb/genny/blob/master/src/workloads/scale/BigUpdate.yml#L9

Reply:
Sure! Thanks for the example. I've included some keywords, let me know if there are any others I should add.